### ISSUE TITLE
Today suddenly error [e] Error 'files'

### ISSUE THREAD

**Issue by Mystefyer** [B16f00t/whapa#162]
Hi, 
Wondering what the error [e] Error 'files' might indicate. Worked fine 2 days ago. I can request the info perfectly. 
When requesting the filelist its lists all files, but in the end throws (I blotted out the rootfolder name):

Traceback (most recent call last):
  File "***\libs\whagodri.py", line 399, in <module>
    for file in wa_backup.backup_files(backup):
  File "***\libs\whagodri.py", line 70, in list_path
    for item in page[last_component]:
KeyError: 'files'

Basically the same when I list the database file. Has the backup failed maybe?

Regards,

Mystefyer

**Comment by Mystefyer** [comment]
Update, after new backup it works again.
